dask_sql/physical/rex/core/call.py

```diff
@@ -600,6 +605,25 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format):
+        # Remove double and single quotes from string
+        format = format.replace('"', "")
+        format = format.replace("'", "")
+
+        if df.dtype == "object":
+            df = pdlike.to_datetime(df)
+            return df.strftime(format)
+        else:
+            df = pdlike.to_datetime(df, unit="s")
+            df = df.strftime(format)
+            result = [timestamp for timestamp in df]
+            return pdlike.Series(result)
+
```

Comment:
qq: is one for string inputs like '2021-12-12' vs the other for integer inputs?

Do we need to do `[timestamp for timestamp in df]`. If we can avoid this we can avoid a host based list for gpu inputs.

Reply:
Yes, the first block is for string '2021-12-12' inputs and the second for numeric inputs.

I can double check about other possible ways besides `[timestamp for timestamp in df]`, but from what I remember extra logic was needed to handle this case.